modules/gitops-release-notes-1-2.adoc

```diff
@@ -0,0 +1,133 @@
+
++
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  labels:
+    example: basic
+spec:
+  sso:
+    provider: keycloak
+  server:
+    route:
+     enabled: true
+----
+* You can now define hostnames using route labels to support router sharding. Support for setting labels on the `server` (argocd server), `grafana`, and `prometheus` routes is now available. To set labels on a route, add `labels` under the route configuration for a server in the ArgoCD CR. 
++
+.Example ArgoCD CR YAML to set labels on argocd server
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  labels:
+    example: basic
+spec:
+  server:
+    route:
+     enabled: true
+     labels: 
+       key1: value1
+       key2: value2
+----
+* The GitOps Operator automatically now grants permissions to Argo CD instances to manage resources in target namespaces by applying labels. Users can label the target namespace with the label `argocd.argoproj.io/managed-by: <source-namespace>`, where the `source-namespace` is the namespace where the argocd instance is deployed. 
+
+[id="fixed-issues-1-2_{context}"]
+== Fixed issues
+The following issues were resolved in the current release:
+
+* Previously, if a user created additional instances of Argo CD managed by the default cluster instance in the openshift-gitops namespace, the application responsible for the new Argo CD instance would get stuck in an `OutOfSync` status. This issue has now been resolved by adding an owner reference to the cluster secret. link:https://issues.redhat.com/browse/GITOPS-1025[GITOPS-1025]
+
+[id="known-issues-1-2_{context}"]
+== Known issues
+These are the known issues in {gitops-title} 1.2:
+
+* When an Argo CD instance is deleted from the source namespace, the `argocd.argoproj.io/managed-by` labels in the target namespaces are not removed. link:https://issues.redhat.com/browse/GITOPS-1228[GITOPS-1228]
+
+* Resource quota has been enabled in the openshift-gitops namespace in {gitops-title} 1.2. This can affect out-of-band workloads deployed manually and workloads deployed by the default Argo CD instance in the `openshift-gitops` namespace. When you upgrade from {gitops-title} `v1.1.2` to `v1.2` such workloads must be configured with resource requests or limits. If there are any additional workloads, the resource quota in the openshift-gitops namespace must be increased.
+
++
+Current Resource Quota for `openshift-gitops` namespace.
++
+[cols="1,1,1",options="header"]
+|===
+| *Resource* | *Requests* | *Limits* 
+
+| CPU
+| 6688m
+| 13750m
+
+| Memory
+| 4544Mi
+| 9070Mi
+
+|===
++ 
+You can use the below command to update the CPU limits.
++
+[source,shell]
+----
+$ oc patch resourcequota openshift-gitops-compute-resources -n openshift-gitops --type='json' -p='[{"op": "replace", "path": "/spec/hard/limits.cpu", "value":"9000m"}]'
```

Comment:
Please use: [source,terminal] the $ signs are rendering in red in the preview: https://deploy-preview-35278--osdocs.netlify.app/openshift-enterprise/latest/cicd/gitops/gitops-release-notes.html#known-issues-1-2_gitops-release-notes